crates/criticalup-core/src/download_server_client.rs

```diff
@@ -117,17 +118,28 @@ impl DownloadServerClient {
         // If the token contains such chars treat the authentication as failed due to an invalid
         // token, as the server wouldn't be able to validate it either anyway.
 
-        // set path to token file for docker
+        // Set path to token file for docker.
         let path_to_token_file = if std::path::Path::new("/.dockerenv").exists() {
             Some("/run/secrets/CRITICALUP_TOKEN")
         } else {
             None
         };
```

Comment:
@Hoverbear Please note that I kept the docker check as-is because it is really tricky to check if one is running inside container using an env var unless we build the image. And I did not want to port this check to an env var somehow.

Reply:
Perfectly fair